README.md

```diff
@@ -18,23 +18,26 @@ const Persist = require('persist-store')
 // custom persisters should implement promises
 
 class LogIt {
-  save(name, value) {
+  save(value) {
     return Promise.resolve('this does nothing')
   }
-  load(name) {
-    return Promise.resolve("shh, act like i loaded something:" + name)
+  load() {
+    return Promise.resolve('shh, act like i loaded something')
   }
 }
 
 // example options
 const persist = new Persist([
     {
       type: 'local',
-      path: './data'
+      path: './data',
+      name: 'file'
     },
     {
       type: 's3',
       bucket: 'a_bucket',
+      prefix: 'some_prefix',
```

Comment:
Ok, I just thought bucket described the top level then prefix was middle level and key was bottom level. I'm cool with this, I'll change it.  (I know it doesn't matter to S3, just thought it was how to describe the terms :))